### PR TITLE
Documentation updating for #1717

### PR DESCRIPTION
**Pull request by manevant**
Added usecase for oscar_fork_app command with dot path for forking to project_root dir and added reference to usecases of oscar_fork_app.

Added in #1717

p.s. Feel free correct my misspellings

### CODE REVIEW

File: docs/source/topics/customisation.rst [hunk 1]
```diff
@@ -49,6 +49,15 @@ you. It is explained in detail in :doc:`fork_app`. Run it like this::
     Creating __init__.py and admin.py
     Creating models.py and copying migrations from [...] to [...]
 
+.. note::
+
+   For forking app in project root directory, call ``oscar_fork_app`` with ``.`` (dot) instead of ``yourproject/`` path.
+   
+   Exapmle:
```
Comment (mvantellingen) [inline docs/source/topics/customisation.rst]: Small typo :-)

File: docs/source/topics/fork_app.rst [hunk 2]
```diff
@@ -8,7 +8,7 @@ This guide explains how to fork an app in Oscar.
 
   The following steps are now automated by the ``oscar_fork_app`` management
   command. They're explained in detail so you get an idea of what's going on.
-  But there's no need to do this manually anymore!
+  But there's no need to do this manually anymore! More information are available in :doc:`/topics/customisation#fork-the-oscar-app`.
```
Comment (mvantellingen) [inline docs/source/topics/fork_app.rst]: "More information *is* available"

**Comment by mvantellingen** [comment]
Thanks, two minor spelling errors

**Comment by manevant** [comment]
Such silly typos :)

**Comment by mvantellingen** [comment]
Thanks! :-)

**Pull request merged**